src/localization/en.json

```diff
@@ -414,6 +414,7 @@
             "export": "Export",
             "reset": "Reset"
         },
+        "custom_style_disable": "Disable Custom Theme everywhere",
```

Comment:
This option should be inverted IMO. It should be "Enable custom profile themes" and default to `true`. The tooltip for custom themes should also be updated to indicate it will show for people who have enabled this feature.

Reply:
Yeah, I agree, it's likely best to avoid negatives, but also I don't think matters much. What tooltip do you mean?